### ISSUE TITLE
Tile System Editor UI and Function like the ShaderForge?

### ISSUE THREAD

**Issue by superfrantv** [Unity-Technologies/2d-extras#296]
Hello,
its possible to create a complete new System, based like the System ShaderForge is using?
so anyone can create very complexe tiles with many different variables, without scripting?

**Comment by ChuanXin-Unity** [comment]
Could you describe more on what you would like from this system?

Would it be something like this: https://learn.unity.com/project/introduction-to-visual-scripting ?

**Comment by superfrantv** [comment]
Yes like the visual scripting feature. The base is tile, then you can define rules with if else statements and so on.

**Comment by ChuanXin-Unity** [comment]
Right now, this visual scripting feature does not seem to work well with ScriptableObjects such as our Tile assets. When ScriptableObjects are better supported, we will definitely look into this!
